app/__init__.py

```diff
@@ -9,11 +9,15 @@ def create_app(config_name='development'):
     app = Flask(__name__)
     app.config.from_object(app_config[config_name])
     app.url_map.strict_slashes = False
-    # salestable()
-    # productstable()
-    # userstable()
+
+    ''' method to create all tables '''
+
+    create_tables()
 
     # Use application to name our blueprint
-    from app.api.v1 import version_1 as v1
-    app.register_blueprint(v1)
+    # from app.api.v1 import version_1 as v1
```

Comment:
Avoid presenting work with commented out lines of code if they are not comments that documents your code

Reply:
Hello Bonifase,
Thank you for this feedback. I am putting it to effect.

Regards
Steven Ennis